src/util/cmdline.h

```diff
@@ -113,6 +113,7 @@ class cmdlinet
   };
 
   void parse_optstring(const char *optstring);
+  bool parse_arguments(int argc, const char **argv);
```

Comment:
:pick: Missing doxygen. Only a nitpick, as it was previously missing from the `parse` member function.